docs/contributing/Building, Debugging, and Testing on Unix.md

```diff
@@ -1,14 +1,87 @@
-# Getting the Code
+# Building, Debugging and Testing on Windows
```

Comment:
>Windows [](start = 37, length = 7)

Linux

Reply:
Hmm ... oops 😦

Reply:
Using `Unix` instead of `Linux` as that's the more genercal name for Mac / Linux testing. The doc has a lot of Linux specific info but it's also generally applicable to Mac as well.